src/Mvc/Mvc.ViewFeatures/src/Rendering/IHtmlHelperOfT.cs

```diff
@@ -70,7 +70,9 @@ public interface IHtmlHelper<TModel> : IHtmlHelper
     /// <paramref name="expression"/> result.
     /// </para>
     /// <para>
-    /// Custom templates are found under a <c>DisplayTemplates</c> folder. The folder name is case-sensitive on
+    /// Custom templates are found under a <c>DisplayTemplates</c> folder within the
+    /// <see href="https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/razor-pages-start#pages-folder-2" >Pages</see > folder.
```

Comment:
```suggestion
    /// <see href="https://docs.microsoft.com/aspnet/core/tutorials/razor-pages/razor-pages-start#pages-folder-2" >Pages</see > folder.
```

That said, we avoid putting direct concept doc links into the API docs because they may change. If we want to link to this resource we'd use an aka.ms link. @mkArtakMSFT is that something you can help with?

Reply:
Here you go: https://aka.ms/aspnet/7.0/razorpages-pages-folder

Note, that I've made this 7.0-specific, as the content may change in the future releases, and we may need to target different pages from the same API docs eventually.